js/test/PixelPass.test.js

```diff
@@ -11,6 +11,14 @@ test("should return decoded data for given QR data", () => {
 });
 
 
+test("should return decoded data for given QR data in cbor", () => {
+    const data = "NCFHPE/Q6:96+963Y6:96P563H0 %2DH0";
+    const expected =  "{\"temp\":15}";
+
+    const actual = decodeData(data);
+    expect(actual).toBe(expected);
+});
+
```

Comment:
please add test in case qr data is not cbor and there is exception while trying to decode with cbor